react/Table/Virtualized/index.jsx

```diff
@@ -28,8 +28,10 @@ const VirtualizedTable = forwardRef(
     },
     ref
   ) => {
-    const [orderDirection, setOrderDirection] = useState('asc')
-    const [orderBy, setOrderBy] = useState(defaultOrder)
+    const [orderDirection, setOrderDirection] = useState(
+      defaultOrder?.direction ?? 'asc'
+    )
+    const [orderBy, setOrderBy] = useState(defaultOrder?.by ?? undefined)
```

Comment:
Why not using only one state to deal with sort and use object as default prop, like you did before? Sorry we misunderstood each other probably.

```
defaultProps = { defaultOrder: {} }

const [order, setOrder] = useState(defaultOrder)
```

Then replace `orderDirection` by `order.direction`, `orderBy` by `order.by` and `setOrderDirection | setOrderBy` in `handleSort` by `setOrder({ by: property, direction: newOrder })`